### PR TITLE
disable forecasted stats

### PR DESCRIPTION
**Pull request by taroface**
DOC-5501
DOC-5493

### CODE REVIEW

**Review by michae2 — APPROVED** [review]
LGTM! Thank you!

File: v22.2/cost-based-optimizer.md [hunk 1]
```diff
@@ -159,6 +157,47 @@ sql_stats_automatic_collection_min_stale_rows = 2000);
...
+    id INT PRIMARY KEY,
+    balance DECIMAL);
+
+ALTER TABLE accounts
+SET (sql_stats_forecasts_enabled = false);
+~~~
+
+The current table settings are shown in the `WITH` clause output of `SHOW CREATE TABLE`:
+
+~~~ sql
```
Comment (kathancox) [inline v22.2/cost-based-optimizer.md:186]: Nit: You can remove the sql syntax highlighting from the output block here. And, the columns are slightly off, small adjustment needed.

Comment (kathancox) [reply v22.2/cost-based-optimizer.md:186]: (I also think there is a output block like this earlier on the page that you could take the opportunity to edit!) 😺

**Review by kathancox — APPROVED** [review]
One tiny nit, but otherwise LGTM!

**Pull request merged**